item/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+from django.contrib.auth.models import User
+# Create your models here.
+
+class Category(models.Model):
+    name = models.CharField(max_length=255)
+
+    class Meta:
+        ordering = ('name',)
+        verbose_name_plural = "Categories"
+
+    def __str__(self):
+        return self.name
+
+class Item(models.Model):
+     category = models.ForeignKey(Category, related_name='items', on_delete=models.CASCADE)
+     name = models.CharField(max_length=255)
+     description = models.TextField(blank=True, null=True)
+     price = models.FloatField()
+     image = models.ImageField(upload_to='item_images', blank=True, null=True)
```

Comment:
an item without image is not very good for selling right? I would personally delete blank and null here

Reply:
price is better to have in DecimalField